src/simulation/simulation.ts

```diff
@@ -76,6 +77,12 @@ export interface AutomationSimulationProps {
    * @default - Uses AWS API to execute the document remotely.
    */
   readonly runCommandHook?: IRunCommandHook;
+
+  /**
+   * Hook for simulating aws:executeAutomation.
+   * @default - Uses AWS API to execute the document remotely.
+   */
+  readonly executeAutomationHook?: IExecuteAutomationHook;
```

Comment:
This is nice - will allow us to plug in the alternate local execution for another local document.